apps/lightning-terminal/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  web:
+    image: lightninglabs/lightning-terminal:v0.3.4-alpha@sha256:ae0b30a73d141bfc24505892b6c23b8faf0acbf93275410aa95118e01f794166
+    user: "1000:1000"
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 5m
+    ports:
+      - "$APP_LIGHTNING_TERMINAL_PORT:$APP_LIGHTNING_TERMINAL_PORT"
+    volumes:
+      - ${APP_DATA_DIR}/lit:/lit
+      - ${APP_DATA_DIR}/faraday:/faraday
+      - ${APP_DATA_DIR}/loop:/loop
+      - ${APP_DATA_DIR}/pool:/pool
+      - ${LND_DATA_DIR}:/lnd:ro
+    command:
+        - --lit-dir="/lit"
+        - --faraday.faradaydir="/faraday"
+        - --loop.loopdir="/loop"
+        - --pool.basedir="/pool"
+        - --insecure-httplisten=0.0.0.0:$APP_LIGHTNING_TERMINAL_PORT
+        - --lnd-mode="remote"
+        - --remote.lnd.network="$BITCOIN_NETWORK"
+        - --remote.lnd.rpcserver=$LND_IP:$LND_GRPC_PORT
+        - --remote.lnd.macaroondir="/lnd/data/chain/bitcoin/$BITCOIN_NETWORK"
+        - --remote.lnd.tlscertpath="/lnd/tls.cert"
```

Comment:
This whole section could be simplified somewhat by storing everything in the same directory:
```
    restart: on-failure
    stop_grace_period: 5m
    ports:
      - "$APP_LIGHTNING_TERMINAL_PORT:$APP_LIGHTNING_TERMINAL_PORT"
    environment:
      - HOME=/home/lit
    volumes:
      - ${APP_DATA_DIR}/lit:/home/lit
      - ${LND_DATA_DIR}:/lnd:ro
    command:
        - --insecure-httplisten=0.0.0.0:$APP_LIGHTNING_TERMINAL_PORT
        - --lnd-mode="remote"
        - --remote.lnd.network="$BITCOIN_NETWORK"
        - --remote.lnd.rpcserver=$LND_IP:$LND_GRPC_PORT
        - --remote.lnd.macaroondir="/lnd/data/chain/bitcoin/$BITCOIN_NETWORK"
        - --remote.lnd.tlscertpath="/lnd/tls.cert"
```

This would mean that the config file in Umbrel should be moved to `apps/lightning-terminal/lit/.lit/lit.conf`.
Of course this is just a suggestion and not necessary. I just noticed that the directory flags are called differently for each project (`loopdir` vs. `basedir` vs. `faradaydir` :grimacing: ) and this would make it a bit more future proof if we ever decide to rename them.

Reply:
Ahh yes great spot!

Also mean we don't need to go creating new dirs and adding new flags if you bundle new daemons into LiT.